app/src/main/java/com/example/itunesapp/repository/Repository.kt

```diff
@@ -0,0 +1,56 @@
+package com.example.itunesapp.repository
+
+import android.content.Context
+import android.util.Log
+import com.example.itunesapp.database.DatabaseImpl
+import com.example.itunesapp.mapper.ResultMappingImpl
+import com.example.itunesapp.model.ResultDatabase
+import com.example.itunesapp.model.SearchItem
+import com.example.itunesapp.network.SearchItunesApi
+import retrofit2.Response
+
+class Repository(
+    private val searchItunesApi: SearchItunesApi,
+    private val databaseImpl: DatabaseImpl,
+    private val applicationContext: Context
+) {
+
+
+
```

Comment:
Let's not add more than one blank lines. You can also take help of android studio's auto format.